site-health.php

```diff
@@ -91,11 +92,12 @@ public function wp_consent_api_test() {
 			if ( $plugins_without_registration ) {
 				$result['status']      = 'recommended';
 				$result['label']       = __( 'One or more plugins are not conforming to the Consent API.', 'wp-consent-api' );
-				$result['description'] = __( 'Not all plugins have declared to follow Consent API guidelines. Please contact the developer.', 'wp-consent-api' );
-				$result['actions']     = implode( '<br>', $not_registered );
+				$result['description'] = __( '<span class="title">Not all plugins have declared to follow Consent API guidelines. Please contact the developer.</span>', 'wp-consent-api' );
```

Comment:
@mujuonly why not move the span outside of the translation function. No point in enabling translators to break that HTML.

Reply:
@garretthyder  Good one

Reply:
@garretthyder  Could you please do it.

Reply:
@mujuonly I created a PR against your branch. If you approve/merge it then it will perculate to this PR.

Reply:
@garretthyder Done (y)